proto/batch_plan.proto

```diff
@@ -197,9 +191,8 @@ message PlanNode {
     DropSourceNode drop_source = 20;
     MergeSortExchangeNode merge_sort_exchange = 21;
     SortMergeJoinNode sort_merge_join = 22;
-    GenerateInt32SeriesNode generate_int32_series = 23;
     HopWindowNode hop_window = 25;
-    GenerateTimeSeriesNode generate_time_series = 26;
+    GenerateSeriesNode generate_series = 26;
```

Comment:
Do the numbers here need to be consecutive or broken

Reply:
They need to be unique, and can be non-consecutive.